formidable/forms/field_builder.py

```diff
@@ -92,6 +93,17 @@ def get_disabled(self):
 
         return False
 
+    def get_initial(self):
+        if self.field_is_dict and 'defaults' in self.field:
+            defaults = self.field['defaults']
+            if len(defaults) > 0:
+                return defaults[0]
```

Comment:
may be you should check if the field is `multiple` ? In this case, return the entire defaults list ?

Reply:
Oh that's right thanks, I will add a test with a multiple field!

Reply:
Done :+1: